src/get-started/test-drive/index.md

```diff
@@ -10,11 +10,24 @@ next:
 toc: false
 ---
 
-This page describes how to create a new Flutter app from templates, run it,
-and experience "hot reload" after you make changes to the app.
+This page describes the following procedures:
```

Comment:
I always feel procedure is an unnecessarily technical word. Consider replacing with something like "tasks" for increased understandability.